subworkflows/local/utils_nfcore_eager_pipeline/main.nf

```diff
@@ -118,10 +118,6 @@ workflow PIPELINE_INITIALISATION {
                 }
             [ meta, r1, r2, bam ]
         }
-        .groupTuple()
```

Comment:
just to make sure - this was the weird input-validation that did nothing and randomly failed?